src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingSubChannel.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.sscs.reference.data.mappings;
+
+import static uk.gov.hmcts.reform.sscs.reference.data.mappings.HearingChannel.*;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum HearingSubChannel {
+
+    TELEPHONE_BT_MEET_ME(TELEPHONE,"telephone-btMeetMe", "Telephone - BTMeetme", "", "BBA3"),
+    TELEPHONE_CVP(TELEPHONE,"telephone-other", "Telephone - Other", "", "BBA3"),
```

Comment:
should be
TELEPHONE_CVP(TELEPHONE,"telephone-cvp", "Telephone - CVP", "", "BBA3"),